Changelog.md

```diff
@@ -12,6 +12,8 @@ Compiler Features:
  * SMTChecker: Support inline arrays.
  * SMTChecker: Support variables ``block``, ``msg`` and ``tx`` in the CHC engine.
  * Control Flow Graph: Print warning for non-empty functions with unnamed return parameters that are not assigned a value in all code paths.
+ * Command Line Interface: New option ``model-checker-engine`` allows to choose a specific SMTChecker engine. Options are ``all`` (default), ``bmc``, ``chc`` and ``none``.
+ * Standard JSON: New option ``settings.modelCheckerEngine`` allows to choose a specific SMTChecker engine. Options are ``all`` (default), ``bmc``, ``chc`` and ``none``.
```

Comment:
We could consider defaulting this to `none` and removed the `pragma experimental SMTChecker;`. So then SMT is always enabled, but not ran by default as there's no engine selected.

Reply:
Would that work when the binary is compiled with Z3 and CVC4 turned off?

Reply:
We could actually do that.
@cameel I guess that would work similar to now when you activate `pragma experimental SMTChecker;` but there's no solver.
The only difference to now would be that it would be enabled for all sources at the same time, whereas now it's enabled only for the sources with the pragma.